.travis.yml

```diff
@@ -82,18 +82,19 @@
 
 language: cpp
 os: linux
+#dist: bionic
```

Comment:
Should probably remove this commented out line.

However, since Travis-CI is seemingly a thing of the past, probably just remove this file after migrating to something else.